app/models/profile.rb

```diff
@@ -0,0 +1,10 @@
+class Profile < ApplicationRecord
+  belongs_to :user
+  has_attached_file :avatar, styles: { thumb: '100x100#' }
+
+  validates_attachment :avatar, content_type: {
+    content_type: ['image/jpeg', 'image/gif', 'image/png']
+  }
+  validates_attachment_file_name :avatar, matches: [/png\z/, /jpe?g\z/]
```

Comment:
`validates_attachment`and `validates_attachment_file_name`is old validations syntax style, use:

```ruby
validates_attachment :avatar, presence: true, content_type: /\Aimage/
```
The filename validation is not needed as it is handeled through `content_type`